src/convert/mapEvent.js

```diff
@@ -39,12 +50,15 @@ const mapEvent = (event) => {
   const keys = event.changes.keys;
   const changes = Array.from(keys.entries(), convertMapOp);
 
-  const baseOp = {
-    type: 'set_node',
-    newProperties: {},
-    properties: {},
-    path: toSlatePath(event.path),
-  };
+  for( var [key, action] of keys.entries()){}
+  
+  var baseOp = {
+      type: (key === 'data' && action.action === 'update' && !event.path.length)?'set_value':'set_node',
```

Comment:
I'm not understanding what 'key' and 'action' are referring to here -- but I'm probably missing something?

Reply:
The key refers to fields that were changed on the operation, and the action refers to the action taken upon the field. These variables were extracted using the block above, 'var' declared inside a block can be accessed outside the block scope. But this logic had critical flaws and was changed.